pkg/docker/helper.go

```diff
@@ -182,6 +191,42 @@ func (r *DockerHelper) FindContainer(labels []string) ([]string, error) {
 	return arr, nil
 }
 
+// FindContainerJSON will manually search for containers with matching labels.
+// This is useful in case the `--filter` doesn't work.
+func (r *DockerHelper) FindContainerJSON(labels []string) ([]string, error) {
+	args := []string{"ps", "-q", "-a"}
```

Comment:
Can we not use --format here to run a single command to find all container labels?